src/common/commonutils/SshUtils.c

```diff
@@ -341,7 +341,7 @@ static int CheckOnlyApprovedMacAlgorithmsAreUsed(const char* macs, char** reason
     FREE_MEMORY(macsValue);
     FREE_MEMORY(sshMacs);
 
-    OsConfigLogInfo(log, "CheckOnlyApprovedMacAlgorithmsAreUsed: %s (%d)", PLAIN_STATUS_FROM_ERRNO(status), status);
+    OsConfigLogInfo(log, "CheckOnlyApprovedMacAlgorithmsAreUsed complete with %d", status);
```

Comment:
It seems that we are sometimes saying "... completed with %d" and here "... complete with %d". Do we want to standardize on one format for this (complete vs completed)?

Reply:
There are two different situations: 
- "complete" (verb, could be also 'completing' but sounds strange) is at the end of current function. 
- "completed" (adjective) is instead of "failed" for calls inside of the function.

Reply:
I'll see what can be changed to avoid such ambiguity

Reply:
Replaced 'complete' with 'returning' and 'completed' with 'returned' for clarity.